Common/Brokerages/DefaultBrokerageModel.cs

```diff
@@ -300,7 +300,7 @@ public virtual IBuyingPowerModel GetBuyingPowerModel(Security security)
                     model = new OptionMarginModel(RequiredFreeBuyingPowerPercent);
                     break;
                 case SecurityType.Future:
-                    model = new FutureMarginModel(RequiredFreeBuyingPowerPercent);
+                    model = new FutureMarginModel(RequiredFreeBuyingPowerPercent, security);
```

Comment:
Are we missing a security argument in `AlphaStreamsBrokerageModel`? It looks similar to the `DefaultBrokerageModel`

Reply:
Also, in the `Future` security class, should we be providing it to the multiple `FutureMarginModel` that we construct?

Reply:
The `BuyingPowerModel` that is set in the `Future` constructor is overridden by the brokerage model.